core/controllers/skill_editor.py

```diff
@@ -96,8 +96,6 @@ def get(self, skill_id):
             'interaction_templates': jinja2.utils.Markup(
                 interaction_templates),
             'dependencies_html': jinja2.utils.Markup(dependencies_html),
```

Comment:
can we remove `,` from here? (I see you have removed this from other places.)

Reply:
Done.